src/utils/getRandomNumberId.ts

```diff
@@ -0,0 +1,9 @@
+export function getRandomNumberId(length = 8): number {
+  let randomId = 0;
+
+  for (let i = 0; i < length; i++) {
+    randomId += Math.floor(Math.random() * 10);
+  }
+
+  return randomId;
+}
```

Comment:
Better to find maxId in the array of todos and to do +1

Reply:
Because now sometimes I get an error on creating todo, because `id` was generated the same as existing one